src/plugins/gs-flatpak.h

```diff
@@ -24,6 +24,7 @@
 #define __GS_FLATPAK_H
 
 #include <gnome-software.h>
+#include <flatpak.h>
```

Comment:
Actually it is needed for `FlatpakProgressCallback`. I'm not 100% sure how this built before, though, but it definitely does not build if I don't add it.